Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -2866,11 +2887,6 @@ RefPtr<CSSValue> ComputedStyleExtractor::propertyValue(CSSPropertyID propertyID,
     if (!styledElement)
         return nullptr;
 
-    if (!isExposed(propertyID, m_element->document().settings())) {
-        // Exit quickly, and avoid us ever having to update layout in this case.
-        return nullptr;
-    }
-
```

Comment:
We need to keep this, but only skip this step when `propertyValue` is called from `copyProperties` (use a boolean argument with a default value so you only need to change one place)